Gemfile

```diff
@@ -55,6 +55,8 @@ gem 'surveyor', :git => 'https://github.com/caboteria/surveyor.git', branch: 'ra
 gem 'therubyracer', '0.10.2', :platforms => :ruby
 gem 'uglifier', '>= 1.0.3'
 gem 'will_paginate'
+gem 'omniauth-cas'
```

Comment:
It looks like this gem is no longer maintained, the last contribution was 2 years ago. You may need to find another solution.
<img width="1126" alt="screen shot 2016-04-15 at 3 25 14 pm" src="https://cloud.githubusercontent.com/assets/1830708/14572731/627f3710-031e-11e6-8a56-6f87b3bfb73e.png">